.github/workflows/release.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         run: |
           IMAGE_NAMESPACE=${{ secrets.QUAYIO_ORG }} VERSION=${{ steps.version.outputs.VERSION }} DOCKER_PUSH=true make image-multi
 
-  bom:
+  Bom:
```

Comment:
few places we use lower case, is this intended?

Reply:
To make all the first letter of github action job name capitalized :)